testbench/common.py

```diff
@@ -894,6 +893,33 @@ def wrapper(*args, **kwargs):
 
     return retry_test
 
```

Comment:
Could you add this instruction to the README as well? Or I guess if the instruction is the same as the reads one, you can update the description to say that this applies to uploads too.

Reply:
Done